dashboard.js

```diff
@@ -0,0 +1,571 @@
+// Third-party dependencies
```

Comment:
Mostly moved from `index.js` but with some changes that I'll try to point out.

Reply:
this is a big step towards simplifying that file!